packages/x-components/src/x-modules/queries-preview/components/__tests__/query-preview.spec.ts

```diff
@@ -101,8 +103,25 @@ describe('query preview', () => {
     expect(getXComponentXModuleName(wrapper.vm)).toBe('queriesPreview');
   });
 
+  it('no sends the `QueryPreviewRequestUpdated` event if saveCache is true, but emits load', () => {
```

Comment:
The descriptions of the tests have to make sense gramatically with the 'it', so it reads like "it tests this thing"

```suggestion
  it('does not send the `QueryPreviewRequestUpdated` event if saveCache is true, but emits load', () => {
```